ops/terraform/env/mgmt/ecr.tf

```
@@ -39,3 +44,93 @@ resource "aws_ecr_repository" "bfd" {
     scan_on_push = true
   }
 }
+
+resource "aws_ecr_registry_policy" "for_bfd" {
+
```

```suggestion
resource "aws_ecr_registry_policy" "this" {
```

Typically we standardize on naming resources `this` (sometimes `main` or `primary`, depending on context). It's not a meaningful name, and its purpose can be easily understood from the surrounding context.

I suggest this resource be renamed (and the below) to `this` to follow the convention.